app/javascript/components/App.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {Route, NavLink, Switch} from 'react-router-dom'
+import Interviews from './Interviews';
+import Interviewers from './Interviewers';
+import Interviewees from './Interviewees';
+import NewInterview from './NewInterview';
+import NewInterviewer from './NewInterviewer';
+import NewInterviewee from './NewInterviewee';
+import EditInterview from './EditInterview';
+
+const App = (props) => {
+  return (
+    <div className="App">
+        
+          <div className="container">
+            <Navigation />
+            <Main />
+          </div>
+       
+      </div>
+  );
+}
+
+const Navigation = () => (
+  <nav className="navbar navbar-expand-lg navbar-light bg-light">
+    <div className="navbar-brand">
+
+       <div className="navbar-item"><NavLink exact className="nav-link"  to="/">Interview App </NavLink></div>
+                        
+    </div>
+               
+    <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
+      <div className="navbar-nav">
+        <div className="nav-item nav-link active"><NavLink exact className="nav-link" activeClassName="active" to="/Interviews/new"> New Interview </NavLink></div>
```

Comment:
The line length is too long. Try to break them

Reply:
80 chars max

Reply:
commit - 832864a